code/__DEFINES/jobs.dm

```diff
@@ -13,6 +13,8 @@
 #define AI_JF				(1<<9)

```

Comment:
I can't comment on the correct line but you need to remove

#define DETECTIVE			(1<<3)

You can put redshield in its place and change the number from 13 to 3